pkg/localstore/gc_test.go

```diff
@@ -732,14 +732,13 @@ func TestGC_NoEvictDirty(t *testing.T) {
 	db := newTestDB(t, &Options{
 		Capacity: 10,
 	})
-
-	closed := db.close
+	defer db.Close()
```

Comment:
This is not needed and also causes closing of a closed channel, as newTestDB already closes the db in its t.Cleanup function.